api/src/config/main.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	"os"
+)
+
+var configMap = map[string]string{}
```

Comment:
not sure why you need this map or putting anything in it.

Reply:
mainly for the github token, I don't wanna read it in 1 place